test/unit_tests/util/messageRendererSpec.js

```diff
@@ -203,9 +203,9 @@ describe('renderMessage', () => {
     );
   });
 
-  it('does not render a broken markdown link', () => {
+  it('escapes links with an xss attempt on their url', () => {
     expect(renderMessage(`[sometext](https://some.domain"><script>alert("oops")</script>)`)).toBe(
-      `[sometext](<a href="https://some.domain" target="_blank" rel="nofollow noopener noreferrer">https://some.domain</a>&quot;&gt;&lt;script&gt;alert(&quot;oops&quot;)&lt;/script&gt;)`,
+      `<a href=\"https://some.domain%22%3E%3Cscript%3Ealert(%22oops%22)%3C/script%3E\" target=\"_blank\" rel=\"nofollow noopener noreferrer\" data-md-link=\"true\" data-uie-name=\"markdown-link\">sometext</a>`,
```

Comment:
This doesn't look like a vulnerability to me, since this link is not valid anyway (try pasting it in search bar), wdyt?

Reply:
@PatrykBuniX Indeed, the intention here is to make sure, that the "injected" html code does not end up in the resulting html.

both (old and new) test results are valid here